cli/resolver.rs

```diff
@@ -177,6 +180,37 @@ impl CliGraphResolver {
   pub fn found_package_json_dep(&self) -> bool {
     self.found_package_json_dep_flag.is_raised()
   }
+
+  fn check_surface_byonm_node_error(
+    &self,
+    specifier: &str,
+    referrer: &ModuleSpecifier,
+    mode: NodeResolutionMode,
+    original_err: AnyError,
+    resolver: &ByonmCliNpmResolver,
+  ) -> Result<(), AnyError> {
+    if let Ok((pkg_name, _, _)) = parse_npm_pkg_name(specifier, referrer) {
+      match resolver
+        .resolve_package_folder_from_package(&pkg_name, referrer, mode)
+      {
+        Ok(_) => {
+          return Err(original_err);
+        }
+        Err(_) => {
+          if resolver
+            .find_ancestor_package_json_with_dep(&pkg_name, referrer)
+            .is_some()
+          {
+            return Err(anyhow!(
+              "Could not resolve \"{}\", but found it in a package.json. Maybe run `npm install`?",
```

Comment:
Should we tell more details here that Deno is currently expecting `node_modules/` directory to be set up by an external package manager?

```
Could not resolve \"{}\", but found it in a package.json. Deno expects `node_modules/` directory to be up to date, did you forget to run `npm/yarn/pnpm install`?
```

Reply:
That sounds good, but I don't think we should put other package managers in the message because users of yarn/pnpm will understand what to do.

Reply:
Agreed!